Vagrantfile

```diff
@@ -117,10 +117,17 @@ Vagrant.configure("2") do |config|
 
 	# Ensure that WordPress can install/update plugins, themes and core
 	if vagrant_version >= "1.3.0"
-		config.vm.synced_folder ".", "/vagrant", :mount_options => [ "dmode=777,fmode=777" ]
-		CONF["synced_folders"].each do |from, to|
-			config.vm.synced_folder from, to, :mount_options => [ "dmode=777,fmode=777" ]
-		end if CONF["synced_folders"]
+		if CONF['virtualbox']['nfs']
```

Comment:
I'd flag it for both. For what it's worth, though I'm using VMware now I used to use VirtualBox a lot in the past and always ran into nasty caching issues with NFS - it's possible that doesn't exist now, but I believe it's something inherent with the OS X file watching with NFS that causes this, not an issue in VMWare.